src/platform/device-manager.ts

```diff
@@ -4,22 +4,21 @@
  **/
 /// <reference types="@google/local-home-sdk" />
 
-import { DeviceManager } from "@google/local-home-sdk";
-
-export class DeviceManagerStub implements DeviceManager{
+import { DeviceManager } from '@google/local-home-sdk';
 
+export class DeviceManagerStub implements DeviceManager {
   markPending(request: smarthome.IntentRequest): Promise<void> {
     //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
+    throw new Error('Method not implemented.');
   }
 
   getProxyInfo(id: string): smarthome.ProxyInfo {
     //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
+    throw new Error('Method not implemented.');
   }
 
-  deviceId: string;
-  error: boolean;
+  deviceId: string = '';
```

Comment:
Do we want the default to be an empty string? Can we leave it null?

Reply:
I added these defaults since tslint was complaining about them never being initialized.  However, since I'm going to implement this full stub in #23, I'm going to remove these variables entirely for now, as well as the `send()` implementation.